libs/components/data-manager/src/lib/modules/data-manager/data-manager-column-picker/data-manager-column-picker.component.ts

```diff
@@ -76,11 +76,6 @@ export class SkyDataManagerColumnPickerComponent implements OnDestroy, OnInit {
   ) {}
 
   public ngOnInit(): void {
-    this.libResources
```

Comment:
We should remove the `private libResources: SkyLibResourcesService` argument from the constructor, too? If not, this will cause an error in the `main` branch due to `noUnusedLocals` being set.